### PR TITLE
Modbus service

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Add description of write_coil.

This is added to "next" because "next" contains a new version of the modbus integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/48633

- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
Please change the target branch to `next` branch.

Or if we're just going to add the docs for the previous service schema to the current branch we need to rebase.

**Comment by janiversen** [comment]
I thought GitHub automatically choose "next" since it is the default branch, but no....I have updated the target branch to next.

I prefer to have it on next together with the rest of the "new configuration".

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -758,12 +758,11 @@ modbus:
 
 ### Services
 
-
 | Service | Description |
```
Comment (MartinHjelmare) [inline source/_integrations/modbus.markdown:763]: Let's add an anchored header for each service to make it easy to link to a specific service.

Comment (janiversen) [reply source/_integrations/modbus.markdown:763]: Anchors are (as far as I know) not standard in markdown, but I have added anchors using GitHub style, I hope that is correct.

Comment (MartinHjelmare) [reply source/_integrations/modbus.markdown:763]: The headers will be anchors by default. See eg:

https://github.com/home-assistant/home-assistant.io/blob/56f6152f33ff66adb7ed3b53ce31e6b0a35bcc14/source/_integrations/zwave_js.markdown#L58

Comment (janiversen) [reply source/_integrations/modbus.markdown:763]: Changed (nice to learn something new).

File: source/_integrations/modbus.markdown [hunk 2]
```diff
@@ -296,7 +296,7 @@ climates:
       default: C
 {% endconfiguration %}
 
-#### Services
+#### Service `modbus.set-temperature`
```
Comment (MartinHjelmare) [inline source/_integrations/modbus.markdown]: Let's keep a general topic Services at level two.
```suggestion
## Services

### Service `modbus.set-temperature`
```

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Review by frenck — APPROVED** [review]

**Pull request merged**